swagger.yaml

```diff
@@ -6221,6 +6225,28 @@ definitions:
       - vsphere
       - none
       - oci
+      - external
```

Comment:
I thought that we are going to remove oci, @avishayt ?

Reply:
I'll come in a follow-up PR, we need to plug the oci code to the external/platformName=oci, then I will be able to the remove oci platform type